### PR TITLE
Add barXtemp.app v1.3.1

### PR DESCRIPTION
**Pull request by MircoT**
Add an useful and free app to get temperature of your mac.

This is a simple OS X app to get the computer temperature. The app is accessible from the Systray and shows the CPU temperature, the integrated GPU temperature (if available), the speed of the fans and the temperature of the battery.

It is a very useful app that you can't find for free (and it is also open, because the project is under mit license and on github).

### CODE REVIEW

File: Casks/barxtemp.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'barxtemp' do
+  version '1.3.1'
+  sha256 'cf9f0da35288f95d19140cb71657aa48ca90c850d43694f851fe27e4b63a8ed2'
+
+  url 'https://github.com/Gabriele91/barXtemp/releases/download/1.3.1/barXtemp.app.zip'
```
Comment (adityadalal924) [inline Casks/barxtemp.rb]: should be `url "https://github.com/Gabriele91/barXtemp/releases/download/#{version}/barXtemp.app.zip"`

Comment (victorpopkov) [inline Casks/barxtemp.rb]: Also, the appcast should be added after the `url`:
```ruby
  url "https://github.com/Gabriele91/barXtemp/releases/download/#{version}/barXtemp.app.zip"
  appcast 'https://github.com/Gabriele91/barXtemp/releases.atom'
```

File: Casks/barxtemp.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  name 'barXtemp'
+  homepage 'http://gabriele91.github.io/barXtemp/'
```
Comment (victorpopkov) [inline Casks/barxtemp.rb]: The HTTPS version of the homepage is available which is more preferred:
```ruby
  homepage 'https://gabriele91.github.io/barXtemp/'
```

**Comment by MircoT** [comment]
I made the changes. Thank you for the suggestions.

File: Casks/barxtemp.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+cask :v1 => 'barxtemp' do
+  version '1.3.1'
+  sha256 'cf9f0da35288f95d19140cb71657aa48ca90c850d43694f851fe27e4b63a8ed2'
+
+  url "https://github.com/Gabriele91/barXtemp/releases/download/#{version}/barXtemp.app.zip"
+  appcast 'https://github.com/Gabriele91/barXtemp/releases.atom'
+
```
Comment (victorpopkov) [inline Casks/barxtemp.rb]: This newline is not needed here.

**Comment by MircoT** [comment]
Done. Sorry, I didn't know...

**Comment by victorpopkov** [comment]
Thank you for your patience in making all those changes. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by MircoT** [comment]
Done... I think. Is ok like that?

**Comment by victorpopkov** [comment]
Thank you for the contribution. It was merged directly as e3df35694b8751c2167a32823ce23f50a597fc44 to keep commit history cleaner. Your contribution is still credited to you.

**Comment by MircoT** [comment]
Thank you for the support and the attention!

**Pull request closed without merging**